src/main/resources/de/nerdclubtfg/signalbot/components/defaultConfig.json

```diff
@@ -2,7 +2,8 @@
   "plugins": {
     "echo": true,
     "fefe" : true,
-    "plugins" : true
+    "plugins" : true,
```

Comment:
Adding a trailing comma would produce nicer diffs.
Con: [This does not conform to JSON.org](http://stackoverflow.com/questions/201782/can-you-use-a-trailing-comma-in-a-json-object) / [RRF4627](http://www.ietf.org/rfc/rfc4627.txt),
but: [FasterXML/Jackson supports trailing commas](https://github.com/FasterXML/jackson-core/issues/118).